pkg/networkmanager/index.html

```diff
@@ -151,6 +161,33 @@ <h2 class="panel-title" translatable="yes">Unmanaged Interfaces</h2>
     </div>
   </div>
 
+  <script id="network-general-settings-template" type="x-template/mustache">
+    <form class="ct-form">
+      <label class="control-label" for="network-general-settings-name-input" translatable="yes">Name</label>
```

Comment:
I don't think we want to expose the connection id at all, much less let people change it.